administration/src/bp-modules/project-settings/UserEndpointSettings.tsx

```diff
@@ -49,7 +49,10 @@ const DeleteIcon = styled(Delete)`
   cursor: pointer;
 `
 
-const UserEndpointSettings = (): ReactElement => {
+type UserEndpointSettingsProps = {
```

Comment:
:upside_down_face: I think it's not "user endoint settings" anymore
ProjectSettings / ApiTokenSettings ?